### ISSUE TITLE
404 not found

### ISSUE THREAD

**Issue by Chensienyong** [bobthecow/genghis#218]
hi guys, i'm kinda new to mongo and using sails.
So the thing is, when i'm opening genghisapp and try to create database from browser, let's say the name is 'test', it shows in table with size n/a.. when i try to open it, it shows

> 404: Database Not Found
> Please try again.

and that database just gone when i return.
The same shows at collection

> 404: Collection Not Found
> Please try again.

**Comment by bobthecow** [comment]
Open up the Network tab of the developer console of your browser, and check to see if there's a non-200 response after creating a database or collection?

**Comment by Chensienyong** [comment]
thanks for fast respond.
nope, when create database or collection it give 200 POST response,
but opening it give 404 GET response.

**Comment by bobthecow** [comment]
Are you using any special characters in the DB (or collection) name?

**Comment by Chensienyong** [comment]
no, i use lowercase letter name, like 'activity', 'user' and other.

**Comment by bobthecow** [comment]
That's strange. Which backend are you using (ruby or php)?

**Comment by Chensienyong** [comment]
how do i know which backend i use?
i install using gem, does that mean i use ruby?

**Comment by bobthecow** [comment]
Yep, that's Ruby. And you're starting it via `genghisapp` on the command line?

**Comment by Chensienyong** [comment]
yup

**Comment by bobthecow** [comment]
Hrm. I've seen similar issues when running behind Apache and using fancy characters in names. I've never run into anything like it with the Ruby backend running from the command line.

Open up a terminal and run `show dbs;` in  `mongo`. Check whether it's actually creating the database and having trouble showing it, or if it's failing to create in the first place.

**Comment by Chensienyong** [comment]
it's fail to create.
everytime i create it shows the database there, but after refresh the page it gone already.
and i tried `show dbs;` right after create database and show only local database which i don't create.
